openassessment/templates/openassessmentblock/staff_area/oa_student_info.html

```diff
@@ -166,7 +166,11 @@ <h2 class="staff-info__title ui-staff__subtitle">
                 </div>
             {% elif workflow_status == "cancelled" %}
                 <div class="staff-info__final__grade__score">
-                    <p>{% trans "The learner's submission has been removed from peer assessment. The learner receives a grade of zero unless you delete the learner's state for the problem to allow them to resubmit a response." %}</p>
+                    {% if not is_team_assignment %}
+                        <p>{% trans "The learner's submission has been removed from peer assessment. The learner receives a grade of zero unless you delete the learner's state for the problem to allow them to resubmit a response." %}</p>
+                    {% else %}
+                        <p>{% trans "The teams's submission has been removed from peer assessment. The team receives a grade of zero unless you delete the team's state for the problem to allow them to resubmit a response." %}</p>
```

Comment:
Team submissions are not part of peer assessments. Perhaps run the wordsmithing by Sapana.

Reply:
Good catch, copy paste strikes again